submissions/Roka20012/pop-up-menu/style.scss

```diff
@@ -169,8 +169,6 @@
     margin: -1px;
     padding: 0;
     border: 0;
-    top: 50px;
-    left: 18px;
 }
```

Comment:
Why your repository contains`.scss` and `.css` files? 
Your PR must contain only source files.

Reply:
Yes it is. I fix it)